### PR TITLE
Add Wyrm support.

### PR DESCRIPTION
**Pull request by Andrei-Dolgolev**
## Changes

Move all dependency from moonstreamdb to `networks` file and remove import of moonstreamdb `models`

### CODE REVIEW

**Review by kompotkot — APPROVED** [review]
lg

**Comment by kompotkot** [comment]
@bugout-dev check
- [ ] First release moonstreamdb with wyrm network

File: moonworm/crawler/networks.py [hunk 1]
```diff
@@ -13,39 +13,17 @@
...
         PolygonLabel,
         PolygonTransaction,
+        WyrmBlock,
+        WyrmLabel,
+        WyrmTransaction,
         XDaiBlock,
         XDaiLabel,
         XDaiTransaction,
     )
+    from moonstreamdb.moonworm import Network, MODELS, tx_raw_types
```
Comment (zomglings) [inline moonworm/crawler/networks.py:23]: It seems a little confusing for me to call the module `moonstreamdb.moonworm`. But I don't have a better name for it.

**Review by zomglings — APPROVED** [review]
lgtm

One comment about `moonstreamdb.moonworm`.

**Comment by Andrei-Dolgolev** [comment]
Coresponding PR in moonstream https://github.com/bugout-dev/moonstream/pull/761

File: moonworm/crawler/networks.py (hunk 1 above)
Comment (kompotkot) [reply moonworm/crawler/networks.py]: networks?

**Pull request merged**